mock-echo-app/src/cmr/mock_echo/client/echo_util.clj

```diff
@@ -478,3 +487,22 @@
          :provider_identity
          {:target "GROUP"
           :provider_id provider-guid}))
+
+(defn grant-permitted?
```

Comment:
I am not sure here is a good place really. I would have preferred to keep them where they are. ;)